hack/build/config.sh

```diff
@@ -23,7 +23,7 @@ CONTROLLER_MAIN="cmd/${CONTROLLER}"
 IMPORTER_MAIN="cmd/${IMPORTER}"
 CLONER_MAIN="cmd/${CLONER}"
 
-DOCKER_IMAGES="cmd/${CONTROLLER} cmd/${IMPORTER} cmd/${CLONER}"
+DOCKER_IMAGES="cmd/${CONTROLLER} cmd/${IMPORTER} cmd/${CLONER} tools/cdi-func-test-file-host-init tools/cdi-func-test-file-host-http"
```

Comment:
Can you extract the `cdi-func-test-file-host-*` names into separate variables

Reply:
yes